scripts/plan.py

```diff
@@ -0,0 +1,70 @@
+from github import Github 
+from github import Auth
+import json
+import os
+import sys
+
+pull_request_id = int(os.getenv("CIRCLE_PULL_REQUEST").split("/")[-1])
+commit_hash = os.getenv("CIRCLE_SHA1")
+token = os.getenv("GITHUB_TOKEN")
+auth = Auth.Token(token)
+
+g = Github(auth=auth)
+
+org_name = "Cyber4All"
+org = g.get_organization(org_name)
+
+repo = g.get_repo("Cyber4All/Security-Injections")
+print(repo)
+
+pullrequests = repo.get_pulls(state='open', sort='updated', base='main')
+pull_request = repo.get_pull(pull_request_id)
+latest_pr = ''
+commits = pull_request.get_commits()
+
+# if pullrequests.totalCount > 0:
+#     # Get the most recent PR
+#     latest_pr = pullrequests[0]
+#     commits = latest_pr.get_commits()
+#     print(f"Latest PR: {latest_pr.title} (#{latest_pr.number})")
+# else:
+#     print("No open pull requests found.")
+#     sys.exit()
+
+absoluteFilePaths = []
+comment = "Modules: "
+
+for c in commits:
+    print(c)
+    files = c.files
+    for f in files:
+        # print("File: " + f.filename)
+        path = os.path.abspath(f.filename)
+        if "content/" in path and path not in absoluteFilePaths:
+            absoluteFilePaths.append(path)
+
+if not absoluteFilePaths: # if no commits have changes to modules, exit
+    print("No changes to modules")
```

Comment:
If you want to add logging to this script for additional info I'm 100% for that because in CI it is very helpful to have verbose logging. In that case you can add python's logging module and configure at the top of the script.

```python3
import logging

logger = logging.getLogger(__name__)

logging.basicConfig(
    level=logging.INFO,
    format="[%(levelname)s] %(asctime)s - %(message)s",
    datefmt="%Y-%m-%d %H:%M:%S",
)

# Example of using logging
logger.info("New Log to output")
logger.error("This is an error")
```

Reply:
I'll Implement this!